packages/adblocker/test/engine/engine.test.ts

```diff
@@ -445,6 +485,51 @@ $csp=baz,domain=bar.com
       });
     });
 
+    it('redirect-rule with priority', () => {
+      const { redirect } = createEngineWithResource(
+        ['||foo.com$image,redirect-rule=foo.js:10', '||foo.com$image'],
+        'foo.js',
+      ).match(request);
+      expect(redirect).to.eql({
+        body: 'foo.js',
+        contentType: 'application/javascript',
+        dataUrl: 'data:application/javascript;base64,Zm9vLmpz',
+      });
+    });
+
+    it('redirect-rule with with no priority wins', () => {
```

Comment:
https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#redirect

> By default, redirect directives have an implicit priority of 0. Filter authors can declare explicitness by appending :[integer] (negative values are also supported) to the redirect= option token.

I read it as `a.js` (priority 10) should win here.